docs/includes/topic_capacity-based-scaleout.rst

```diff
@@ -0,0 +1,112 @@
+:orphan: true
+
+Capacity-Based Scale Out
+========================
+
+Overview
+--------
+
+When using :ref:`differentiated service environments <Differentiated Service Environments>`, you can configure capacity metrics for the F5® agent to provide scale out across multiple BIG-IP device groups. The F5 agent :ref:`configuration parameters <Configure the F5 OpenStack Agent>`  ``environment_group_number`` and ``environment_capacity_score`` allow the F5 LBaaSv2 agent scheduler to assign requests to the group that has the lowest capacity score.
+
+Each F5 agent is first configured, via the ``icontrol_endpoint`` parameter, to manage a BIG-IP device or device group. Agents with the same ``environment_group_number`` manage the same BIG-IP device group, regardless of whether they are configured with the same iControl® endpoints. This means that an agent can be configured to manage a specific device, but also be available to handle requests for its group.
```

Comment:
This isn't correct.  We should try to try to describe the environment_group_number as a way to map a number to a group of icontrol_endpoints.  It's just an ID for a set of endpoints.  If two agents are in the same environment_group, then they should have the same set of endpoints.  The number is just a convenient way of identifying them.

"This means that an agent can be configured to manage a specific device, but also be available to handle requests for its group."

The only way for us to be able to manage devices is to have the correct icontrol_endpoints.  So if a group comprises 3 devices with different IP addresses, every agent within the group needs to have all the endpoints in order to be in that group.